src/main/webapp/script.js

```diff
@@ -8,4 +10,12 @@ function getRandomImage() {
     gallery.src = IMAGE_ARRAY[result];
 
     console.log(result, IMAGE_ARRAY[result]);
+}
+
+function addServletText() {
+    fetch('/data').then((response) => response.text()).then(response_text => {
+        document.getElementById('fetch_test').innerHTML = response_text;
+        console.log(response_text);
```

Comment:
If the console.log calls are only used for your personal debugging, I'd recommend removing them before merging your code to master.